### ISSUE TITLE
Tom & Jerry: House Trap (Europe) - Broken pause menu in Software renderer

### ISSUE THREAD

**Issue by LoStraniero91** [duckstation-unofficial/issue-tracker#127]
Duckstation 0.1 - 6292 - g5d3cf93a

http://redump.org/disc/11475/

![image](https://github.com/duckstation-unofficial/issue-tracker/assets/29802222/1fe17859-405c-4b29-9547-cdf4211ed922)

Tested with all enhancements disabled, software renderer at native resolution. Pause menu is broken in software renderer, but it's displayed correctly in HW renderers.

![image](https://github.com/duckstation-unofficial/issue-tracker/assets/29802222/41aa4dfe-cb32-4bbc-805c-03dd9f5654fb)

HW renderer (Vulkan) comparison:

![image](https://github.com/duckstation-unofficial/issue-tracker/assets/29802222/5050c120-5a6b-4270-855f-726b87e6f2a5)

**Comment by sigarettenstank** [comment]
@LoStraniero91 Yeah I can confirm. Thanks for the report. I wouldn't have noticed that. 

![tj](https://github.com/duckstation-unofficial/issue-tracker/assets/95235094/2e68dbb0-8145-4eb8-affd-6caa6c6cf229)

**Comment by thelastangryman1907** [comment]
@LoStraniero91 This looks like it's fixed now.

![Image](https://github.com/user-attachments/assets/baf0a507-1a90-4371-b668-369ae726ed1f)
